coalib/output/Logging.py

```diff
@@ -1,10 +1,26 @@
 from datetime import datetime
+from collections import defaultdict
 import json
 import io
 import logging
 import logging.config
 
 
+class CounterHandler(logging.Handler):
+    """
+    A logging handler which counts the number of calls
+    for each logging level.
+    """
+    level_called = None
+
+    def __init__(self, *args, **kwargs):
+        super(CounterHandler, self).__init__(*args, **kwargs)
+        CounterHandler.level_called = defaultdict(lambda: 0)
```

Comment:
Yes, that would work, but I don't understand your issue with the resetting of the counter when an instance of the counter is created. The instance is create once for a run, when the logging is set up. Isn't resetting the counter then exactly what we want?